src/qibo/tests/test_parallel.py

```diff
@@ -12,7 +12,7 @@
 
 def test_parallel_circuit_evaluation(backend):
     """Evaluate circuit for multiple input states."""
-    if 'GPU' in qibo.get_device() or os.name == 'nt': # pragma: no cover
+    if 'GPU' in qibo.get_device() or os.name == 'nt' or qibo.get_backend() == "tensorflow": # pragma: no cover
```

Comment:
Optionally we could also cover the error cases here using:
```Python
with pytest.raises(RuntimeError):
    ...
```
instead of `pytest.skip`. This way we could remove `# pragma: no cover` from the errors above and include them in the coverage check.

Reply:
I was thinking about it, however then we need to wrap the parallel function call in a if/else condition, right? (and duplicate the test basically)

Reply:
Yes, I am not sure if it is the best way but this is what I do in a few other tests where I check for similar errors. For these tests I think it is sufficient to do something like:
```Python
if 'GPU' in qibo.get_device() or os.name == 'nt' or qibo.get_backend() == "tensorflow":
    with pytest.raises(RuntimeError):
        r2 = parallel_execution(c, states=states, processes=2)
else:
    r2 = parallel_execution(c, states=states, processes=2)
    np.testing.assert_allclose(r1, r2)
```
so the code duplication is not that large. Not very necessary to do this though.

Reply:
Yeah, let's keep as it is, thanks.